db/migrate/20221026151206_add_users_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Same here, Please add the foreign key between posts and users tables as `author_id`, not `user_id` as mentioned on the ERD table.